BoxContentSDK/BoxContentSDK/Protocols/BOXContentCacheClientProtocol.h

```diff
@@ -213,6 +213,8 @@
                               withFolder:(BOXFolder *)folder
                                    error:(NSError *)error;
 
+- (void)finishFolderItemsRequest:(BOXFolderItemsRequest *)request;
```

Comment:
Minor: naming, hasFinishedFolderItemsRequest? Just to make it clear that it isn't a call to action, but rather information that an event has occurred.